server/src/main/java/org/opensearch/index/mapper/MappedFieldType.java

```diff
@@ -304,7 +304,7 @@ public Query prefixQuery(
     }
 
     // Case sensitive form of wildcard query
-    public final Query wildcardQuery(String value, @Nullable MultiTermQuery.RewriteMethod method, QueryShardContext context) {
+    public Query wildcardQuery(String value, @Nullable MultiTermQuery.RewriteMethod method, QueryShardContext context) {
         return wildcardQuery(value, method, false, context);
     }
```

Comment:
Can this go back to being `final`?

Reply:
I missed this
Thanks for pointing this out